pages/common/ac.md

```diff
@@ -0,0 +1,11 @@
+# ac
+
+> A faster version of cat that can strip ANSI color codes.
+
+- Dump a file's contents:
+
+`ac file`
+
+- Strip color codes from output of some command:
+
+`curl -s wttr.in/LA | ac -s`
```

Comment:
Same here. Perhaps change that url to example.com or something and wrap it in `{{` and `}}`?